dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/AbstractClusterInvoker.java

```diff
@@ -333,10 +351,15 @@ protected LoadBalance initLoadBalance(List<Invoker<T>> invokers, Invocation invo
 
 
     private void setContext(Invoker<T> invoker) {
+        setContext(invoker, null);
+    }
+
+    private void setContext(Invoker<T> invoker, URL consumerUrl) {
         RpcContext context = RpcContext.getServiceContext();
         context.setInvoker(invoker)
-                .setRemoteAddress(invoker.getUrl().getHost(), invoker.getUrl().getPort())
-                .setRemoteApplicationName(invoker.getUrl().getRemoteApplication());
+            .setRemoteAddress(invoker.getUrl().getHost(), invoker.getUrl().getPort())
+            .setRemoteApplicationName(invoker.getUrl().getRemoteApplication())
```

Comment:
needless to set remote-address and remote-application-name

Reply:
ok，i will remove it.